programmers/난이도별/level01.하샤드_수/rockmiin.py

```diff
@@ -0,0 +1,12 @@
+def solution(x):
+    answer=True
+    sum=0
+    for i in str(x):
+        sum+=int(i)
+    if x%sum!=0:
+        answer=False
```

Comment:
```suggestion
        return False
```
여기서 바로 return 하는게 좋을 것 같아요~